pkg/synthetictests/disruption.go

```diff
@@ -16,6 +16,24 @@ import (
 	"k8s.io/kubernetes/test/e2e/framework"
 )
 
+// isExcludedDisruptionBackend returns true if any of the given backends are in the
+// disruption backend name.  Essentially, we want to test these disruption backends
+// but flake them for now to avoid failing payloads.
+func isExcludedDisruptionBackend(name string) bool {
+	excludedNames := []string{
+		"ci-cluster-network-liveness",
+		"kube-api-http1-external-lb",
```

Comment:
why is this excluded?

Reply:
It's a new backend, anything mentioning http is, the originals were left alone and are running along side these.